src/frontend/src/flows/manage.ts

```diff
@@ -75,9 +112,23 @@ const pageContent = (userNumber: bigint, devices: DeviceData[]) => html`
     <label>Identity Anchor</label>
     <div class="highlightBox">${userNumber}</div>
     <div class="labelWithAction">
-      <label id="deviceLabel">Added devices</label>
-      <button class="labelAction" id="addAdditionalDevice">
-        ADD NEW DEVICE
+      <label class="labelAddedDevices" id="deviceLabel"
+        ><span class="tooltip"
+          >You can register up to ${MAX_AUTHENTICATORS} authenticator devices
+          (recovery devices excluded)</span
+        >Added devices
+        (${numAuthenticators(devices)}/${MAX_AUTHENTICATORS})</label
```

Comment:
amount of data is small so perf are good but `numAuthenticators` is iterated twice. just in case you rather like to use a state that is assigned once.

Reply:
Right, but I couldn't figure out a way to save that in a variable without turning the whole `html` bit into a full blown function with body; that'll do